plugins/module_utils/network/common/utils.py

```diff
@@ -667,7 +675,7 @@ def remove_empties(cfg_dict):
     return final_cfg
 
 
-def validate_config(spec, data):
+def validate_config(spec, data, module=None):
```

Comment:
I think we can add `redact` parameter as well and update no_log_values when `redact` is set to `True` and `module` parameter is passed